clients/unshaded/src/main/java/tachyon/client/FileSystemMasterClient.java

```diff
@@ -55,6 +52,7 @@
    */
   public FileSystemMasterClient(InetSocketAddress masterAddress, TachyonConf tachyonConf) {
     super(masterAddress, tachyonConf);
+    mCompatibleVersions.add(Constants.FILE_SYSTEM_MASTER_SERVICE_VERSION);
```

Comment:
Is it possible to set all clients up in a central place, other than distributed in each client?